README.md

```diff
@@ -100,5 +99,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
-
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- **[OPTIONAL]** Kindly correct the path to the LICENSE file to make sure that all the links work as expected for professional documentation 👉🏽 ` [MIT](./LICENSE)`